notebooks/gauss_plot.ipynb

```diff
@@ -0,0 +1,149 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    random_var = norm
</pre><p>Let's not do this. We can use <code>norm</code> directly. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/712/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>